tableauserverclient/models/flow_run_item.py

```diff
@@ -0,0 +1,106 @@
+import xml.etree.ElementTree as ET
+from ..datetime_helpers import parse_datetime
+import itertools
+
+
+class FlowRunItem(object):
```

Comment:
is this a type of JobItem? It looks very similar, might be nice not to have several almost-identical job items.

Reply:
It is indeed pretty similar (`started_at`, ...) but also has some unique properties (`flow_id`, ...)
Not sure what to do about this...
Maybe we could factor out those common fields into a base class - but then again: is it worth it?

@jacalata do you have some particular solution in mind here?